styles/styles.css

```
@@ -0,0 +1,459 @@
+@font-face {
+    font-family: 'PT Sans';
+    src: url("../fonts/ptsans-700.woff2") format("woff2");
+    font-weight: 700;
+    font-style: normal;
+    font-display: swap;
+}
+
+@font-face {
+    font-family: 'PT Sans', sans-serif;
```

В `@font-face` не нужно указывать альтернативный шрифт. Сейчас шрифт 400 не подключен.
```suggestion
    font-family: 'PT Sans';
```